### PR TITLE
Add opacity/tint to Tiled Sprite 

### PR DESCRIPTION
**Pull request by Wend1go**
Action, Condition and Expression

### CODE REVIEW

File: Extensions/TiledSpriteObject/Extension.cpp [hunk 1]
```diff
@@ -29,6 +29,47 @@ void DeclareTiledSpriteObjectExtension(gd::PlatformExtension& extension) {
 #if defined(GD_IDE_ONLY)
   obj.SetIncludeFile("TiledSpriteObject/TiledSpriteObject.h");
 
+  obj.AddCondition("GetOpacity",
```
Comment (4ian) [inline Extensions/TiledSpriteObject/Extension.cpp]: Can we name this "Opacity"? New convention is "XXX" for conditions (no Get), "SetXXX" for actions, "XXX" for expressions (like conditions).

**Comment by 4ian** [comment]
You're on fire 🔥🔥 Thanks for this! Looks mostly good to me, just added a comment, will merge then.

**Comment by Wend1go** [comment]
Ready
Will add the same functions to the 9-Patch Sprite after merging.

**Comment by 4ian** [comment]
I've moved the SetFunctionName to JsExtension.cpp
(Think of Extension.cpp as "declaring" the extension for GDevelop editor, while JsExtension.cpp is about bridging from GDevelop to JavaScript by setting the function names, and the optional getter function name) (In JsExtension.js this is mixed in the same file, but old extensions like Text objects are supposed to be working in C++ too, hence the two files).

Waiting for confirmation that this is compiling (I did the changes online) and merging then!

**Comment by Wend1go** [comment]
```
/home/travis/build/4ian/GDevelop/Extensions/TiledSpriteObject/Extension.cpp:68:8: error: 
      no member named 'SetIncludeFile' in 'gd::ExpressionMetadata'
      .SetIncludeFile("TiledSpriteObject/TiledSpriteObject.h");
       ^
/home/travis/build/4ian/GDevelop/Extensions/TiledSpriteObject/Extension.cpp:81:8: error: 
      no member named 'SetIncludeFile' in 'gd::InstructionMetadata'
      .SetIncludeFile("TiledSpriteObject/TiledSpriteObject.h");
       ^
2 errors generated.
```

**Pull request merged**